presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -213,7 +219,8 @@ public void printFinalInfo()
     private void printQueryInfo(QueryResults results)
     {
         StatementStats stats = results.getStats();
-        Duration wallTime = nanosSince(start);
+        long now = System.nanoTime();
+        Duration wallTime = succinctNanos(now - start);
```

Comment:
@ausgoo  I see your point. I have no strong opinions either way. Let's keep the current version and see if @raghavsethi has any thoughts :)